### DISCUSSION TITLE
Is there going to ever be a way here to use the wyzeapy addon without needing HA

### DISCUSSION THREAD

**Discussion by sideup66** [SecKatie/wyzeapy#80] [General]
Hi is there any chance there will be documentation for this repository again? Id love to use it on my system without HA, however without documentation its kind of hard to understand how to properly use the script or deal with authentication into the wyze system. Namely, im looking to control motion sensing on my camera using this api in a simple python script. Any help on that would be appreciated. I know this is kind of a support question, but I appreciate the help in advance. 

Thanks!

**Comment by sideup66** [comment]
Update as I was able to piece together my own python script looking at the unit tests in the source here, but it may be helpful to have docs. Id be happy to help if this was needed!

**Reply by FeatherKing** [reply]
could you share your python script in a gist? I am trying to work out some contributions to this repo and seeing your work might speed me along

**Reply by sideup66** [reply]
Sure thing, see here! https://gist.github.com/sideup66/1517adf98fba6db55cfc5f1b68a8cb5f

In the whole its pretty simple. I used the loop from the loop test which was kinda lazy but I literally have one camera, so it doesnt matter that I dont iterate for a specific device. 

the wrapper bash I call it with isnt anything too exciting, just some email and sms logic using my carrier's mail relay and a retry if an error comes back from python for whatever reason. Let me know if you need anything else.

**Reply by FeatherKing** [reply]
thank you! similar to what i started doing. working for me!
